session_4/blog/app/views/articles/_carousel.html.erb

```diff
@@ -0,0 +1,38 @@
+<div id="carouselExampleCaptions" class="carousel slide" data-ride="carousel">
+  <ol class="carousel-indicators">
+    <li data-target="#carouselExampleCaptions" data-slide-to="0" class="active"></li>
+    <li data-target="#carouselExampleCaptions" data-slide-to="1"></li>
+    <li data-target="#carouselExampleCaptions" data-slide-to="2"></li>
+  </ol>
+  <div class="carousel-inner" >
```

Comment:
Instead of hardcoding this as `a1`, `a2` and `a3`, see if you iterate over an array of features articles.

Reply:
[https://github.com/hrushikeshj/IRIS-RoR-Bootcamp-2020/blob/session_4_hrushikesh/session_4/blog/app/views/articles/_carousel.html.erb](https://github.com/hrushikeshj/IRIS-RoR-Bootcamp-2020/blob/session_4_hrushikesh/session_4/blog/app/views/articles/_carousel.html.erb)